cmd/bosun/sched/sched.go

```diff
@@ -704,6 +725,111 @@ func (a ActionType) MarshalJSON() ([]byte, error) {
 	return json.Marshal(a.String())
 }
 
+type Incident struct {
+	Id       uint32
+	Start    time.Time
+	End      time.Time
+	AlertKey expr.AlertKey
+}
+
+func (s *Schedule) createIncident(ak expr.AlertKey, start time.Time) *Incident {
+	s.incidentLock.Lock()
+	defer s.incidentLock.Unlock()
+	if s.Incidents == nil {
+		s.Incidents = map[uint32]*Incident{}
```

Comment:
I'm aware that Go has 2 ways of initing memory. But we use make more than `{}`. Could we continue to use make everywhere?